docs/modules/ROOT/pages/how-tos/testing/build/index.adoc

```diff
@@ -1 +1,27 @@
-= Build-time tests
\ No newline at end of file
+= Build-time tests
+
+This document covers the build-time tests (formerly known as "sanity tests") that {ProductName} runs as part of its component build pipeline. These build-time tests automatically check all application images to ensure that they're up-to-date, correctly formatted, and protected from security vulnerabilities.
+
+The component build pipeline for {ProductName} include various types of build-time tests, conducted using link:https://tekton.dev/docs/pipelines/tasks/#overview[Tekton tasks]. For validating container information, the pipeline employs https://www.conftest.dev/[Conftest]. The table below outlines the currently implemented build-time tests:
+
+.Deprecated image checks
+|===
+|Test name |Description |Failure message
+
+|image_repository_deprecated |Deprecated images are no longer maintained, leading to unresolved security vulnerabilities. | The container image must not be built from a repository  marked as 'Deprecated' in COMET
+|===
+
+.Unsigned RPM check
+|===
+|Test name |Description |Failure message
+
+|image_unsigned_rpms |Packages signed with Red Hat's secure signing server adheres to stringent policies and procedures. |All RPMs in the image must be signed. Found following unsigned rpms(nvra):
+|===
+
+
+[role="_additional-resources"]
+== Additional resources
+
+* Snyk test is also available to run at build time in the default pipelines.For more information about enabling Snyk task, see xref:how-tos/testing/build/snyk.adoc[Enabling a Snyk task].
```

Comment:
This task is only special by how it is enabled. I think it can still go into the table above.

Reply:
seems reasonable for now.